tests/CsvFileErrorsTest.php

```diff
@@ -40,7 +43,7 @@ public function invalidFileNameProvider()
     {
         return [
             ["", 'Filename cannot be empty'],
-            ["\0", 'fopen() expects parameter 1 to be a valid path, string given'],
+//            ["\0", 'fopen() expects parameter 1 to be a valid path, string given'],
```

Comment:
co s timhle, smazat nebo pridat komentar proc je to zakomentovany?

Reply:
v php8 sa to chová trochu jinak a nejde to moc rozumně vyřesit bez hacků a ztráty dalšího času pro knihovnu co je obsolete.

Reply:
ok